types/jquery/index.d.ts

```diff
@@ -25,14 +25,14 @@
 // TypeScript Version: 2.3
 
 declare module 'jquery' {
-    function factory(window: Window): JQueryStatic;
+    function factory(window: Window): JQueryStatic & JQuery;
```

Comment:
Here's a repro for the bug mentioned in #17298. `jQuery` should represent `JQueryStatic` in that scenario but is `factory` instead. During testing, I _thought_ I had it so that it was importing only one of the two types based on the import form.
- `factory` when using `import jqueryFactory = require('jquery');`
- `JQueryStatic` when using `import * as jQuery from 'jquery';`

I haven't been able to reproduce that behavior. (And no, it didn't make sense to me why it was working.)